src/main/java/com/patchworkmc/mapping/TargetMappingsHolder.java

```diff
@@ -0,0 +1,91 @@
+package com.patchworkmc.mapping;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import net.fabricmc.tinyremapper.IMappingProvider;
+
+import net.patchworkmc.manifest.accesstransformer.v2.exception.MissingMappingException;
+
+import com.patchworkmc.manifest.converter.accesstransformer.AccessTransformerConverter;
+
+/**
+ * A lazily loading representation of all Minecraft names and descriptors in the target mappings.
+ * Used in {@link AccessTransformerConverter} to resolve descriptors for fields and resolve all names for wildcards.
+ */
+public class TargetMappingsHolder {
```

Comment:
I think a name like `MemberInfo` would probably be more indicative of what this is doing, since the purpose of this class is not directly related to mappings / remapping